src/veh_interact.cpp

```diff
@@ -1564,6 +1576,8 @@ void veh_interact::display_stats()
         }
     }
 
+    bool is_boat = !veh->all_parts_with_feature(VPFLAG_FLOATS).empty();
+
     fold_and_print(w_stats, y[7], x[7], w[7], c_ltgray,
                    _("K aerodynamics: <color_ltblue>%3d</color>%%"),
```

Comment:
Should be percentages for presentation to the user. A food cart having traction of 16% and a tank 89% is a reasonable UI